src/components/footer/Footer.tsx

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable prettier/prettier */
+import React, { useEffect, useState } from 'react';
+import { Todo } from '../../types/Todo';
+
+interface Props {
+  todos: Todo[];
+  filterStatus: (status: string) => void;
+}
+
+export const Footer: React.FC<Props> = ({ todos, filterStatus }) => {
+  const [countCompletedTodo, setCountCompletedTodo] = useState(0);
+  const [statusFilter, setStatusFilter] = useState<
+  'all' | 'active' | 'completed'
+  >('all');
+
+  useEffect(() => {
+    const completedCount = todos.filter(todo => !todo.completed).length;
+
+    setCountCompletedTodo(completedCount);
+  }, [todos]);
```

Comment:
```suggestion
  const completedCount = todos.filter(todo => !todo.completed).length;
```
it can be just the simple var